packages/lib/server/repository/eventType.ts

```diff
@@ -192,11 +207,97 @@ export class EventTypeRepository {
           ...where,
         },
         select,
+        cursor: cursor ? { id: cursor } : undefined,
+        take: limit ? limit + 1 : undefined, // We take +1 as itll be used for the next cursor
         orderBy,
       });
     }
   }
 
+  static async findTeamEventTypes({
+    teamId,
+    parentId,
+    userId,
+    limit = 10,
```

Comment:
```suggestion
    limit = 10,
```

This isn't breaking anything as it is a new function but I would prefer limit to be passed explicitly as it could cause the same problem where only 10 records are being returned and user isn;t aware of it.